bundles/org.eclipse.passage.lbc.server/src/org/eclipse/passage/lbc/server/jetty/JettyServer.java

```diff
@@ -12,32 +12,30 @@
  *******************************************************************************/
 package org.eclipse.passage.lbc.server.jetty;
 
-import java.util.Map;
-
 import org.eclipse.jetty.server.Server;
-import org.eclipse.passage.lbc.server.Port;
-import org.osgi.service.component.annotations.Component;
+import org.eclipse.passage.lbc.server.i18n.Messages;
 
-@Component
 public final class JettyServer {
 
 	private Server server;
 
-	@SuppressWarnings("restriction")
-	public void launch(Map<String, Object> arguments) {
+	public void launch(Port port) {
 		try {
-			server = new Server(new Port(arguments).get().get());
+			server = new Server(port.get());
+			server.setHandler(new Handler());
 			server.start();
-		} catch (Exception e) {
-			e.printStackTrace();
+			System.out.println(String.format(Messages.started, port.get()));
+		} catch (Exception exception) {
+			System.err.println(String.format(Messages.error_onstart, exception.getClass(), exception.getMessage()));
```

Comment:
No debug prints, please

Reply:
Is SLF4J a good alternative for the case?

Reply:
switched to jetty logger